keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -234,54 +234,77 @@ implementation.  The "Design Details" section below is for the real
 nitty-gritty.
 -->
 #### Terminology
+- **multicluster namespace** - A placeholder name for a group of
+  Kubernetes namespaces with the same name on different clusters.
+  This API specifies the semantics for sharing services
+  amongst the namespaces that belong to a multicluster namespace.
+
+  Implementations of this API are responsible for defining and tracking
+  multicluster namespaces and their membership. The specific mechanism
+  is out of scope of this proposal.
 
 - **clusterset** - A placeholder name for a group of clusters with a high
   degree of mutual trust and shared ownership that share services amongst
   themselves. Membership in a clusterset is symmetric and transitive. The set
   of member clusters are mutually aware, and agree about their collective
-  association. Within a clusterset, [namespace sameness] applies and all
-  namespaces with a given name are considered to be the same namespace.
-  Implementations of this API are responsible for defining and tracking
-  membership in a clusterset. The specific mechanism is out of scope of this
-  proposal.
+  association. Within a clusterset, [namespace sameness] applies such that all
+  namespaces with a given name are considered members of a same-named
+  *multicluster namespace*.
+
+  > Note: Clusterset is an optional, high-trust model for defining multicluster
+  namespaces. Implementations are not required to implement clustersets.
+  Instead, they may provide other means for defining multicluster namespaces, for
+  example by allowing directional, pair-wise federation between same-named
+  namespaces of mutually-untrusting clusters.  However, the clusterset model is
+  easy to reason about and a natural model for implementors to target.
```

Comment:
This to me is the core of the edit. We don't want to remove ClusterSet from the spec. MCS is defined based on ClusterSets and I don't think we want to change that. What we do want to say is that it is valid to implement the MCS API outside of ClusterSets, but that not all features may work as expected and you'll need to understand what that means for your chosen implementation - e.g. does namespace sameness only apply selectively? Do all clusters receive the import?